bindings/wasm/examples-account/README.md

```diff
@@ -0,0 +1,37 @@
+![banner](./../../../.meta/identity_banner.png)
+
+
+## IOTA Identity Account Examples
+
+This folder provides code examples for you to learn how the IOTA Identity WASM bindings for the Account can be used in JavaScript/Typescript.
+
+The examples are written in Typescript and can be run independently with Node.js. 
```

Comment:
The idea is to make a clean, minimal project without the need for webpack configuration to avoid complexity for people just starting to learn about the framework (and the account). Also, this allows auto-completion without the need for extra configuration (on VS Code for example). This format is open for change in a future issue, but it is outside the scope of this PR.

Reply:
If it's outside the scope of this PR then this PR should follow the existing conventions and match the existing Webpack examples.

Reply:
This PR will be merged into an epic branch, this issue will be dealt with before merging into dev. See #304.